faqs/galaxy/histories_transfer_entire_histories_from_one_galaxy_server_to_another.md

```diff
@@ -27,7 +27,7 @@ At the **sender** Galaxy server
 At the **receiver** Galaxy server
 
 8. Confirm that you are logged into your account.
-9. Click on **User** in the top menu, and choose **Histories** to reach your **Saved Histories**.
+9. Click on **Data** in the top menu, and choose **Histories** to reach your **Saved Histories**.
```

Comment:
i guess we're going to have to go back and "fix" all of these again in next release when the menu moves to the activity bar

Reply:
Yes, a whole group of navigations will split into masthead + activity bar. Fun! :) Maybe we can come up with some simple text that can be reused. Or, maybe a new FAQ just about navigation/UI landmarks.